pkg/apis/migration/v1alpha1/migplan_types.go

```diff
@@ -23,8 +23,8 @@ import (
 
 // MigPlanSpec defines the desired state of MigPlan
 type MigPlanSpec struct {
-	SrcClusterRef         *kapi.ObjectReference `json:"srcClusterRef,omitempty"`
-	DestClusterRef        *kapi.ObjectReference `json:"destClusterRef,omitempty"`
+	MigSrcClusterRef      *kapi.ObjectReference `json:"migSrcClusterRef,omitempty"`
```

Comment:
Guessing this is here to disambiguate between a reference to a cluster-registry `Cluster` and a `MigCluster`, but I think we'll want to keep `MigCluster` in there as one word, otherwise it looks like we're just prepending `Mig` to everything in our spec (_in my opinion_), and ambiguity remains of whether we are referring to a `Cluster` or `MigCluster`.

Ideas
 - `SrcMigClusterRef`
 - `MigClusterSrcRef`

Reply:
Oh, yeah, completely agree.  I was not paying close enough attention when I changed that.  I think `SrcMigClusterRef` etc would be most consistent.

Reply:
Fixed